content/tutorials/spatial_standards_vector/index.Rmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "How to use open vector file formats in R: GeoPackage & GeoJSON"
+description: "A simple tutorial to demonstrate the use of *.gpkg and *.geojson files in R"
+author: "Floris Vanderhaeghe"
+date: 2019-11-19
+categories: ["r", "gis"]
+tags: ["gis", "r"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE
+)
+unlink(c("sac.*", "habitatstreams.*"))
+```
+
+
+This tutorial uses a few basic functions from the [dplyr](https://dplyr.tidyverse.org) and [sf](https://r-spatial.github.io/sf/) packages.
+While only a few functions are used, you can use the previous hyperlinks to access the tutorials (vignettes) of these packages for more functions and information.
+
+```{r warning=FALSE, message=FALSE}
+options(stringsAsFactors = FALSE)
+library(tidyverse)
+library(sf)
+library(inborutils)
+```
+
+You find a bit more background about 'why and what', regarding the below open standards, in [a separate post](../../articles/geospatial_standards/) on this website.
+
+In short, the GeoPackage and GeoJSON formats are ideal for exchange, publication, interoperability & durability and to open science in general.
+
+## How to make and use GeoPackages (`*.gpkg`)
+
+### Making a GeoPackage from a geospatial `sf` object in R
+
+As an example, we download a geospatial layer of Special Areas of Conservation in Flanders (version _sac_2013-01-18_) from Zenodo:
+
+```{r}
+# meeting a great function from the 'inborutils' package:
+download_zenodo(doi = "10.5281/zenodo.3386815")
+```
+
+_Did you know this: you can visit a website of this dataset by just prefixing the DOI string ^[DOI = Digital Object Identifier. See <https://www.doi.org>.] by `doi.org/`!_
+
+The data source is a shapefile, in this case consisting of 6 different files.
+Read the geospatial data into R as an `sf` object, and let's just keep the essentials (though it doesn't matter for the GeoPackage):
+
+```{r}
+sac <- 
+  read_sf("sac.shp") %>% 
+  select(sac_code = GEBCODE,
+         sac_name = NAAM,
+         subsac_code = DEELGEBIED,
+         polygon_id = POLY_ID)
+```
+
+Have a look at its contents by printing the object:
+
+```{r paged.print=FALSE}
+sac
+```
+
+To write the GeoPackage, we just use the GPKG driver of the powerful [GDAL](https://gdal.org) library (supporting most open and some closed formats), which can be elegantly accessed through `sf::st_write()`:
+
+```{r}
+sac %>% 
+  st_write("sac.gpkg")
+```
+
+Is that all?\
+_**YES :-)**_
+
+Really?\
+_**YES :-)**_
+
+Well, hmmm, if you really want to know a little bit more...
+
+A GeoPackage can contain many layers.
```

Comment:
Would love to see this demonstrated as well. Maybe even in combination with a raster?

Reply:
Let's try.

Reply:
This has been added in d5aa961